doc/classes/SpriteFrames.xml

```diff
@@ -56,22 +57,35 @@
 			<return type="float" />
 			<param index="0" name="anim" type="StringName" />
 			<description>
-				The animation's speed in frames per second.
+				Returns the animation's speed in frames per second.
 			</description>
 		</method>
-		<method name="get_frame" qualifiers="const">
-			<return type="Texture2D" />
+		<method name="get_frame_count" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="anim" type="StringName" />
+			<description>
+				Returns the number of frames in the animation.
+			</description>
+		</method>
+		<method name="get_frame_duration" qualifiers="const">
+			<return type="float" />
 			<param index="0" name="anim" type="StringName" />
 			<param index="1" name="idx" type="int" />
 			<description>
-				Returns the animation's selected frame.
+				Returns the animation's selected frame relative duration (default [code]1.0[/code]).
+				You can calculate the absolute duration (in seconds) of a frame using the following formula:
```

Comment:
```suggestion
				Returns a relative duration of the frame [param idx] in the [param anim] animation (defaults to [code]1.0[/code]). You can calculate the absolute duration (in seconds) of a frame using the following formula:
```

New lines create paragraphs, which is excessive here. And it's good to refer to the parameters by name.

Also, relative to what? I think that information is missing, even though with the formula this can probably be deduced.

Reply:
I was inspired by examples from `AStarGrid2D`.

https://github.com/godotengine/godot/blob/dc3175e4cdb4e54e1c6200b6c4d07646eea95284/doc/classes/AStarGrid2D.xml#L164-L172

I tried replacing `codeblock` with `code`, but it didn't save much space (only one line after `[/codeblock]`, but I guess it's an EditorHelp bug). Other options look even worse.

> Also, relative to what?

Roughly speaking, this is the duration relative to other frames, but more precisely it is the duration relative to some conventional unit (since there can be no frame in the animation with duration of `1.0`). Perhaps "base duration" is a better term than "relative duration"?

Last time, this terminological issue stopped this PR for several months, despite the fact that there were no comments left on other issues. So if we don't have some happy idea to solve this, I suggest we leave it as it is.

Reply:
> I tried replacing `codeblock` with `code`, but it didn't save much space (only one line after `[/codeblock]`, but I guess it's an EditorHelp bug). Other options look even worse.

Codeblock is correct. What I was saying is that you had a new line between the first two sentences. I removed it in my suggestion, and explained why it was not needed. The rest of the formatting is fine.

> Roughly speaking, this is the duration relative to other frames

That would be enough to clarify IMO.

Reply:
I added the following example: `For example, a frame with a duration of [code]2.0[/code] is displayed twice as long as a frame with a duration of [code]1.0[/code].`.